courses/management/commands/gen_fake_data.py

```diff
@@ -0,0 +1,51 @@
+"""
+Generates fake data
+"""
+from random import randint
+from django.core.management import BaseCommand
+
+from courses.factories import CourseFactory, ModuleFactory
+
+
+class Command(BaseCommand):
+    """
+    Generates fake data
+    """
+    help = "Generates fake data"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--courses',
+            dest='courses',
+            default=3,
+            help='Number of courses to generate',
+        )
+
+        parser.add_argument(
+            '--modules',
+            dest='modules',
+            default=5,
+            help='Number of modules to generate',
+        )
+
+        parser.add_argument(
+            '--random',
+            action='store_true',
```

Comment:
How would someone make this argument false? Can you change this to `--not-random` or something to that effect, or remove it entirely?